### PR TITLE
README file updated

### PR DESCRIPTION
**Pull request by parameshwarbhat411**
README.md file updated with reqirements, key-features and deliverables

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,28 @@
 # GitHub Gems: Driving Open-Source Investments With Data
 
-Welcome to the GitHub Gems project! This project hosts a data analytics pipeline that enables smarter investment decisions by measuring the popularity of open-source repos on Github.
+## Project Title and Description
```
Comment (edsioufi) [inline README.md]: Minor nit-pick: Maybe rename this to "Overview" and remove the mention of "project" and "driving open-source investments" to make it look less academic?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
+
+- **Popularity Metrics:** Tracks the growth rate of repositories based on stars and forks to gauge overall popularity.
+- **Contribution Metrics:** Assesses contributions through commit activities and pull request rates to understand developer involvement.
+- **Influence Score:** A composite metric that integrates various signals such as stars, forks, and issues to evaluate the project's influence.
+- **Developer Engagement Index:** Measures the frequency and recency of contributions, indicating active development and project health.
+- **Issue Resolution Efficiency:** Analyzes how quickly issues are resolved, offering insights into the project's operational efficiency.
+- **Dependency Risk Analysis:** Examines the number and reliability of project dependencies to assess stability risks.
+- **Innovation Rate:** Monitors the introduction of new features and technologies, identifying projects that are at the forefront of innovation.
+- **Funding and Sponsorship Activity:** Tracks sponsorship activities to gauge financial support and commercial interest in the project.
```
Comment (edsioufi) [inline README.md:20]: This list of metrics is great – but kinda overpromises 😂 
How about a "further work" section?

**Review by edsioufi — CHANGES_REQUESTED** [review]
Awesome job! Really like how you've summarized the key points and made the README look like a real-life product. Dropped some suggestions to push it even futher.

File: README.md (hunk 2 above)
Comment (parameshwarbhat411) [reply README.md:20]: Resolved and updated the readme file with the fact and dimension models

**Review by edsioufi — APPROVED** [review]
Great job! 🙌

**Pull request merged**